site/src/constants.js

```diff
@@ -15,6 +15,20 @@ export const auth0 = {
   salt: SALT,
 }
 
+export const cluster = {
+  apiUrl: CLUSTER_API_URL,
+  basicAuthToken:
```

Comment:
we need to add these as secrets right ? if yes we should update readme instructions